salt/modules/win_groupadd.py

```diff
@@ -100,36 +103,29 @@ def add(name, **kwargs):
             The name of the group to add
 
     Returns:
-        dict: A dictionary of results
+        bool: ``True`` if successful, otherwise ``False``
 
     CLI Example:
 
     .. code-block:: bash
 
         salt '*' group.add foo
     '''
-    ret = {'name': name,
-           'result': True,
-           'changes': [],
-           'comment': ''}
-
     if not info(name):
-        compObj = _get_computer_object()
+        comp_obj = _get_computer_object()
         try:
-            newGroup = compObj.Create('group', name)
-            newGroup.SetInfo()
-            ret['changes'].append('Successfully created group {0}'.format(name))
-        except pywintypes.com_error as com_err:
-            ret['result'] = False
-            if len(com_err.excepinfo) >= 2:
-                friendly_error = com_err.excepinfo[2].rstrip('\r\n')
-            ret['comment'] = 'Failed to create group {0}. {1}' \
-                             ''.format(name, friendly_error)
+            new_group = comp_obj.Create('group', name)
+            new_group.SetInfo()
+            log.debug('Successfully created group {0}'.format(name))
+        except pywintypes.com_error as exc:
+            msg = 'Failed to create group {0}. {1}'.format(
+                name, win32api.FormatMessage(exc.excepinfo[5]))
+            log.debug(msg)
```

Comment:
Is `debug` the right log level for an error like this?
```suggestion
            log.error(msg)
```